src/network/Connection.ts

```diff
@@ -200,7 +214,7 @@ export class DirectWriter extends Writer {
     }
 
     close(): void {
```

Comment:
I am not sure but this method different syntax than abstract method? Do we need to change this method also?

Reply:
we don't need to, because the parameter will be unused. It does not matter. I added it.